MinStack.java

```diff
@@ -0,0 +1,65 @@
+// Time Complexity : O(1) as everything takes constant time.
+// Space Complexity : O(2n) as I have used two stacks
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this : this statement s.peek().equals(minS.peek()) took so much time. I was using == to compare the values.
+//had to refer to https://stackoverflow.com/questions/52224835/stack-peek-comparison-returns-wrong-result-for-1024 to solve the problem
```

Comment:
try to be familiar with the most useful functions for collections, it'll be of great use in the future